plugins/modules/content_view.py

```diff
@@ -170,6 +188,7 @@ def main():
             description=dict(),
             label=dict(),
             composite=dict(type='bool', default=False),
+            rolling=dict(type='bool', default=False),
```

Comment:
I guess the API defaults to false if nothing is sent, right? Then you can omit it here which will allow us to skip re-recording VCR and won't trigger "empty" updates when the users run this the first time.

Reply:
I guess, yes.
However, I would like to add a test for the option. However, it looks like I not only have to re-record the VCR-cassette but also update /tests/fixtures/apidoc/katello.json :-/
Is there a command to this like with the VCRs?